docs/Building.md

```diff
@@ -4,7 +4,7 @@ If for some reason you require a non-release version of Cromwell or are developi
 features or fixes, the following are required to build Cromwell from source:
 
 * [Scala 2.12.2](http://www.scala-lang.org/news/2.12.1#scala-212-notes)
```

Comment:
In a similar vein there are two different patch level versions of Scala called out here but neither is the 2.12.4 version we're actually using. 😢